### PR TITLE
Update macros to refer to all onboarded chains

### PR DESCRIPTION
**Pull request by belen-pruvost**
Follow up from [this comment](https://github.com/duneanalytics/arrakis-jobs/pull/2749#issuecomment-1665630676)
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

File: macros/public/all_evm_chains.sql [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {% macro all_evm_chains() %}
-   {{ return(array('ethereum', 'optimism', 'arbitrum', 'avalanche_c', 'polygon', 'bnb', 'gnosis', 'fantom')) }}
+   {{ return(array('ethereum', 'optimism', 'arbitrum', 'avalanche_c', 'polygon', 'bnb', 'gnosis', 'fantom', 'base', 'bitcoin', 'celo', 'goerli', 'solana')) }}
```
Comment (jeff-dude) [inline macros/public/all_evm_chains.sql:2]: ah, this is `all_evm_chains`, so i don't think we'd include bitcoin/solana? not sure about celo

Comment (jeff-dude) [reply macros/public/all_evm_chains.sql:2]: the other three macros are fine, but this could cause impact downstream in spellbook

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: macros/public/all_evm_chains.sql (hunk 1 above)
Comment (jeff-dude) [reply macros/public/all_evm_chains.sql:2]: i would just ignore this file in this PR.

@MSilb7 is updating these macros here: https://github.com/duneanalytics/spellbook/pull/3931/files

Comment (belen-pruvost) [reply macros/public/all_evm_chains.sql:2]: AH, you're right - I should not include them on DuneSQL either

Comment (belen-pruvost) [reply macros/public/all_evm_chains.sql:2]: I'll drop the changes for it here.

**Comment by belen-pruvost** [comment]
@jeff-dude I'm removing solana and bitcoin from the function on DuneSQL. Should I also remove goerli (testnet) and celo? Or should those be added by @MSilb7 on https://github.com/duneanalytics/spellbook/pull/3931/files?

**Comment by jeff-dude** [comment]
> @jeff-dude I'm removing solana and bitcoin from the function on DuneSQL. Should I also remove goerli (testnet) and celo? Or should those be added by @MSilb7 on https://github.com/duneanalytics/spellbook/pull/3931/files?

we may want to wait to alter the functions on dunesql until we finalize that linked PR. seems like a minor disconnect currently. having a separate testnet macro is going to be important for how downstream spells are built and use the macro in dbt.

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**